src/modules/constants/constants.service.ts

```diff
@@ -0,0 +1,44 @@
+import { Injectable, InternalServerErrorException, Logger, NotFoundException } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+
+import { DbResponseHelper } from '../../helpers/db-response.helper';
```

Comment:
should this come from an alias path? i.e. `@ukef/helpers/...`

Reply:
Updated